package.json

```diff
@@ -8,6 +8,15 @@
   "main": "dist/a11y-dialog.js",
   "module": "dist/a11y-dialog.esm.js",
   "types": "dist/a11y-dialog.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/a11y-dialog.esm.js",
+      "browser": "./dist/a11y-dialog.js",
+      "require": "./dist/a11y-dialog.cjs",
+      "types": "./dist/a11y-dialog.d.ts"
+    },
+    "./*": "./*"
```

Comment:
I like doing this: just exposing all files by default, as-typed. This is safer than having to manually type out every file one-by-one (and newer files don’t get exposed if they’re not typed out).

IMO trying to be clever, and trying to anticipate how people have manually-imported specific files from `/dist/`, can backfire unless you’ve tested every possible setup. Instead, I like to trust that users do the right thing, and if they’re manually importing a file, just let them.